src/rmq/rmqa/rmqa_connectionimpl.cpp

```diff
@@ -53,7 +53,7 @@ createRetryHandler(const bsl::shared_ptr<rmqio::TimerFactory>& timerFactory,
                    const rmqt::Tunables& tunables)
 {
     if (tunables.find("IIR") != tunables.end()) {
-        bsl::cout << "WARNING: Infinite Immediate Retry Set" << bsl::endl;
```

Comment:
Worth noting that this has always intentionally printed to stdout as a means to highlight and discourage this behaviour since it turns off any backing off on retries.

We may want to keep this behaviour but we can discuss in the team

Reply:
May leave a comment that it's intended then